### PR TITLE
adds a template for requesting VMs

### LINKED ISSUE #5128: Add a template for requesting new VMs
See https://github.com/pulibrary/pul-it-handbook/blob/main/services/new_vms_for_devs.md for some pointers. Help devs make sure their requests are complete and actionable.

### PR DESCRIPTION
**Pull request by acozine**
Closes #5128.

Creates a template for requesting new VMs. This is not meant to be a required form. It's more of a place to gather whatever information the requester has about a request for new resources.

Builds on [existing documentation](https://github.com/pulibrary/pul-it-handbook/blob/main/services/new_vms_for_devs.md).

### CODE REVIEW

**Review by maxkadel — APPROVED** [review]
Thanks @acozine!

File: .github/ISSUE_TEMPLATE/new_vm.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+- [ ] Amount of memory per VM (in GB):
+- [ ] Amount of storage/disk space per VM (in GB):
+
+What firewall changes should be made? (for example, port 22 for SSH, etc.)
+- [ ] Port number(s)
+- [ ] Service(s) 
+
+What shared resources should be mounted? (for example, Isilon drives)
+- [ ] Resources to mount:
+
```
Comment (rlskoeser) [inline .github/ISSUE_TEMPLATE/new_vm.md:33]: Is it worth including a question whether VMs will need access to the NFS server?

**Review by rlskoeser — COMMENTED** [review]
This will be useful, thank you!

File: .github/ISSUE_TEMPLATE/new_vm.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+name: Request a new VM
+about: Assemble the necessary information to request a new VM
+title: ''
+labels: ''
+assignees: ''
+
+---
+### Describe the VM(s) you need
+
+If you do not know the answers to some of these questions, that's okay. Please add the information you do have, and we will work together to gather any other information we need.
```
Comment (rlskoeser) [inline .github/ISSUE_TEMPLATE/new_vm.md:11]: you could also add a link to the new vm documentation for more details (unless that is going away)

**Comment by acozine** [comment]
Also need to update the link in https://github.com/pulibrary/pul-it-handbook/blob/main/services/new_vms_for_devs.md so it opens this type of issue.

File: .github/ISSUE_TEMPLATE/new_vm.md (hunk 1 above)
Comment (acozine) [reply .github/ISSUE_TEMPLATE/new_vm.md]: Good idea, thanks. I will broaden the scope of that last question to futureproof it, and add NFS server access as an example.

File: .github/ISSUE_TEMPLATE/new_vm.md (hunk 2 above)
Comment (acozine) [reply .github/ISSUE_TEMPLATE/new_vm.md]: good idea, added in 36f24aaeb1cdcd84a553c340a0393e6409c17de1

**Review by maxkadel — APPROVED** [review]
I still approve! Thank you @acozine!

**Pull request merged**